index.php

```diff
@@ -0,0 +1,29 @@
+<?php
+    include "includes/header.php";
+
+    //write PHP here...
+    $weight = '225';
+    $age = '43';
+    $first_name = 'Tony';
+    $last_name = 'Mainelli';
```

Comment:
since you are using global vars in your function - i'd declare a $kg placeholder here equal to 0.